src/dynadojo/systems/gilpin_flows.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+from ..abstractions import AbstractSystem
+import importlib
+import warnings
+import os
+import json
+import dysts
```

Comment:
This dependency is missing in `pyproject.toml`

Please add a new optional dependencies section called "gilpin" and include any new dependencies. Then add the "gilpin" group to the "all" group. You should commit the pyproject.toml, but not your lockfile.